src/platform/site-wide/sass/shame.scss

```diff
@@ -1,6 +1,16 @@
 $medium-phone-screen: 375px;
 /* OVERRIDE */
 .merger {
+
+  // Begin new-grid override
```

Comment:
The `new-grid` class will make the overrides easier to find, and therefore, easier to delete. That's why I'm not using utilities.